telco-examples/edge-clusters/dhcp/dual-stack/single-node/README.md

```diff
@@ -0,0 +1,82 @@
+
+- `bmh-example_v4dhcp-v6ra+dhcp_v4v6dns.yaml` - this configuration is identical to the `bmh-example_v4dhcp-v6ra+dhcp.yaml` above, with the exception of statically provided DNS servers; it can be useful in networks where only RAs are avaiable but with no DNS option.
+     Requires:
+         - `${DNS_SERVER_V4}` and/or `${DNS_SERVER_V6}` - the IP address(es) of the DNS server(s) to use, as a single or multiple entries; both IPv4 and/or IPv6 addresses can be specified
+
+**Note**:
+ * Due to limitations in [nmstate](https://nmstate.io/devel/yaml_api.html#ipv6-autoconf), it is not possible to enable SLAAC but disable DHCPv6; for this reason DHCP will always be enabled in the provided examples.
+ * The network configuration will only apply to the provisioned BMH and not yet on the inspection phase of the host to gather hardware information. This is due to a bug that will force the use of DHCP and RA for IPv4 and IPv6 respectively with the default settings; a resolution is espected in the coming weeks.
+
+Once the required BMH file has been selected and refined, in order for it to work, it must also be filled with some values common to all the BMH definitions in addition to the network specific ones previously mentioned. For more information on these values or on how to enroll a Bare Metal Host, please refer to the main Edge Cluster documentation for single host deployments [here](https://github.com/suse-edge/atip/tree/main/telco-examples/edge-clusters#example-1---deploy-a-single-node-edge-cluster-with-the-image-generated-and-telco-profiles).
+
+
+## Cluster definitions
+
+The CAPI cluster definitions to be applied once the bare metal host has been inspected and is available for provisioning, are organized into two directories, depending on whether a single CNI plugin is sufficient or if multihomed Pods are required, through the use of the [Multus](https://github.com/k8snetworkplumbingwg/multus-cni) meta-plugin. Note that the use of the SR-IOV plugin along with Multus is of course possible but not covered, as it doesn't require any specific IPAM configuration.
+
+The definition files come with any CNI plugin specific configuration that may be required and are organized as follow:
+
+- `single-cni`:
+    - `cluster-v4v6-calico.yaml` - the cluster will use Calico as the sole CNI plugin in the cluster
+    - `cluster-v4v6-canal.yaml` - the cluster will use Canal as the sole CNI plugin in the cluster
+    - `cluster-v4v6-cilium.yaml` - the cluster will use Cilium as the sole CNI plugin in the cluster
+
+- `multus-cni`:
+   - `cluster-v4v6-calico_hostlocal.yaml` - the cluster will use Calico as the CNI plugin in combination with "host-local" as Multus IPAM for the second interface
```

Comment:
This could be a table.

Reply:
Yes, I thought of that, and I will do it, but honestly I don't see the value in having the "Telco" prefix in a ATIP, that is telco specific, repo. Also, some example are not necessarily telco specific in the end. I think it's clearer to have filenames based on the roles (as they are all "CAPI" anyway) BMH & Cluster. What do you think? We can align them for now and change them afterward?

Reply:
Okay, will do that.